.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,24 @@
-<!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
-
+<!-- READ ME FIRST: Please fill explanation section below and check off every point from the Essential Checklist! -->
 ## Explanation
 <!--
   - Explain what your PR does. If this PR fixes an existing bug, please include
   - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
   - when this PR is merged.
   -->
 
-## Checklist
+## Essential Checklist
 <!-- Please tick the relevant boxes by putting an "x" in them. -->
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+- [ ] The PR title and explanation starts with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
+- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have rationale included in the PR description.
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
-- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
-- [ ] The PR is made from a branch that's **not** called "develop".
-- [ ] The PR is made from a branch that is up-to-date with "develop".
-- [ ] The PR's branch is based on "develop" and not on any other branch.
-- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.
+- [ ] The PR does not contain any unnecessary code changes from Android Studio.
+- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
+- [ ] The PR is **assigned** to the appropriate reviewers ([reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)).
+
+## For UI-specific PRs only
+<!-- Delete these section if this PR does not include UI-related changes. -->
+If your change includes PR changes, then:
+- Add a video showing the full UX flow with a screen reader enabled (see [accessibility guide](https://github.com/oppia/oppia-android/wiki/Accessibility-(A11y)-Guide))
+- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes (see screenshot guide)
```

Comment:
...portrait/landscape for both the before & after UI changes in both a tablet and phone.

(Missing link to screenshot guide?)

Reply:
Suggest moving this to be the first point since having a screenshot/video guide will be useful for all of the items I think.

Reply:
Done.